packages/knex/src/schema/SchemaGenerator.ts

```diff
@@ -382,6 +386,7 @@ export class SchemaGenerator {
     Utils.runIfNotEmpty(() => col.index(indexName), index);
     Utils.runIfNotEmpty(() => col.unique(uniqueName), prop.unique);
     Utils.runIfNotEmpty(() => col.defaultTo(prop.defaultRaw ? this.knex.raw(prop.defaultRaw) : null), !sameDefault);
+    Utils.runIfNotEmpty(() => col.comment(columnComment), columnComment);
```

Comment:
this should be enough (if its falsy, we won't use the value anyway)

```suggestion
    Utils.runIfNotEmpty(() => col.comment(prop.comment), prop.comment);
```

Reply:
comment(value: string): ColumnBuilder;

it fail there, because knex expect a string cannot accept undefined, so what should i do?

Reply:
that is just TS warning, using `prop.comment!` should work

Reply:
ah okay, I just want to follow your convention